sunpy/net/dataretriever/tests/test_swap.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import pytest
+
+from sunpy.time.timerange import TimeRange
+from sunpy.net.vso.attrs import Time,Instrument,Level
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+import sunpy.net.dataretriever.sources.swap as swap
+
+LCClient = swap.SWAPClient()
+
+@pytest.mark.online
+@pytest.mark.parametrize("timerange,url_start,url_end, level",
+                         [(TimeRange('2015/12/30 00:00:00','2015/12/30 23:59:59'),
+                           'http://proba2.oma.be/swap/data/bsd/2015/12/30/swap_lv1_20151230_000044.fits',
+                           'http://proba2.oma.be/swap/data/bsd/2015/12/30/swap_lv1_20151230_235935.fits', 1)])
+def test_get_url_for_time_range(timerange, url_start,url_end, level):
+    urls = LCClient._get_url_for_timerange(timerange, level = 1)
+    assert isinstance(urls, list)
+    assert urls[0] == url_start
+    assert urls[-1] == url_end
+
+def test_can_handle_query():
+    ans0 = swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'),Instrument('swap'), a.Level(1))
+    assert ans0 == True
```

Comment:
```_can_handle_query``` has been written in a way that it will always return a boolean type, so I don't see why it would return any type apart from boolean. Regardless, I have updated the file implementing your suggestions.